canvas_modules/harness/src/client/App.js

```diff
@@ -33,14 +33,14 @@ import { jsPDF } from "jspdf";
 import * as htmlToImage from "html-to-image";
 
 import { getMessages } from "../intl/intl-utils";
-import * as HarnessBundles from "../intl/locales";
+import HarnessBundles from "../intl/locales";
 import CommandActionsBundles from "@elyra/canvas/locales/command-actions/locales";
 import CommonCanvasBundles from "@elyra/canvas/locales/common-canvas/locales";
 import CommonPropsBundles from "@elyra/canvas/locales/common-properties/locales";
 import PaletteBundles from "@elyra/canvas/locales/palette/locales";
 import ToolbarBundles from "@elyra/canvas/locales/toolbar/locales";
 
-import { CommonCanvas, CanvasController, CommonProperties, ColorPicker } from "common-canvas"; // eslint-disable-line import/no-unresolved
+import { CommonCanvas, CanvasController, CommonProperties, ColorPicker } from "@elyra/canvas";
```

Comment:
Is this importing the correct module?  Does HMR still work?

There is a config in webpack that determines this name.